commown_devices/tests/test_project_task.py

```diff
@@ -154,6 +154,43 @@ def test_ui_repair(self):
         lot_names = set(choices["lot_id"].mapped("name"))
         self.assertEqual(lot_names, {"cc2", "cc3"})
 
+    def test_wizard_involved(self):
+
+        self.task.contract_id = self.c1
+        self.task.lot_id = self.task.contract_id.quant_ids[0].lot_id
+
+        values, possible_values = self.prepare_ui(
+            "project.task.involved_device_picking.wizard", self.task, "task_id"
+        )
+
+        self.assertEqual(
+            values["present_location_id"],
+            self.task.lot_id.quant_ids.filtered(
+                lambda q: q.quantity > 0
+            ).location_id.id,
+        )
+
+        # Create a picking
```

Comment:
Sur ce commentaire j'ajouterais "and check the lot location at the end of the picking".